app/classes/Typolib/Typolib/Rule.php

```diff
@@ -150,6 +154,94 @@ public static function displayIfThenArrayRule()
     }
 
     /**
+     * Check a "if x then y" rule (just for ellipsis character)
+     * TODO : generic method for any character of the ifThen rule array
+     *
+     * @param string      $userString the string entered by the user
+     * @param string      $search     the searched character in $userString to replace
+     * @param string enum $mode       the mode used by the user (list of errors or text corrected)
+     */
+    public function checkIfThenRuleEllipsis($userString, $mode)
+    {
+        $mystring = "Yes ... that's true, but ...";
+        $search = '/\.{3}/'; // equivalent to ...
+        $search2 = '...';
+        $replace = '&#8230;'; // ellipsis character
```

Comment:
We should use UTF-8 character (…) here, while it will work for strings displayed on web pages, it won't work for software strings.